### DISCUSSION TITLE
Dore remark-parse support streaming now?

### DISCUSSION THREAD

**Discussion by yueqing-wolai** [remarkjs/remark#975] [Q&A] [answered]
I found that micromark is supporting read markdown by stream from their github website. I wonder if remark-parse support streaming right now, I didn't find any documentation of it, if it is supported, can anyone tell me how to use it or where is the documentation?

**Comment by wooorm** [comment, marked as answer]
No, remark does not support streams. Streaming has only overhead for ASTs.

You could use unified as a stream already though: https://github.com/unifiedjs/unified-stream.
